clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/config/AwsProviderConfig.groovy

```diff
@@ -71,21 +67,21 @@ class AwsProviderConfig {
       new AwsProvider(accountCredentialsRepository, Collections.newSetFromMap(new ConcurrentHashMap<Agent, Boolean>()))
 
     synchronizeAwsProvider(awsProvider,
-                           amazonCloudProvider,
-                           amazonClientProvider,
-                           accountCredentialsRepository,
-                           objectMapper,
-                           discoveryApiFactory,
-                           eddaApiFactory,
-                           ctx,
-                           registry,
-                           reservationReportScheduler)
+      amazonCloudProvider,
```

Comment:
would prefer not to see code reformatting unrelated to the change at hand in PRs